boot/bootutil/include/bootutil/image.h

```diff
@@ -85,17 +85,17 @@ struct flash_area;
 #define IMAGE_TLV_PUBKEY            0x02   /* public key */
 #define IMAGE_TLV_SHA256            0x10   /* SHA256 of image hdr and body */
 #define IMAGE_TLV_RSA2048_PSS       0x20   /* RSA2048 of hash output */
-#define IMAGE_TLV_ECDSA224          0x21   /* ECDSA of hash output */
-#define IMAGE_TLV_ECDSA256          0x22   /* ECDSA of hash output */
```

Comment:
I wonder what's gonna happen if the image generation tool uses these TLVs. Not everybody uses `imgtool`, yet.

Reply:
I've put them back and marked them as in #1641